docs/auraed/index.md

```diff
@@ -21,7 +21,7 @@ We suggest using the [aurae](https://github.com/aurae-runtime/aurae) repository
 If you intend on building this repository directly you can leverage the Makefile in this repository.
 
 ```bash
-make
+make auraed-debug
```

Comment:
just `auraed` :)

Reply:
My bad. 🙈.

I updated it in the files I deleted, but had already copy-pasted it. 😐 

Oh wait, no it's something different from that, because I was also trying to move it somewhere else, so I don't think that's the reason.